tests/Aql/SortedCollectExecutorTest.cpp

```diff
@@ -104,29 +105,39 @@ class SortedCollectExecutorTestNoRowsUpstream : public ::testing::Test {
 };
 
 TEST_F(SortedCollectExecutorTestNoRowsUpstream, producer_doesnt_wait) {
```

Comment:
Is this test actually relevant?
I if so we shall rename it
```suggestion
TEST_F(SortedCollectExecutorTestNoRowsUpstream, producer_gets_empty_input) {
```